TechnicalDocuments/Proposals/Fish_Operator.md

```diff
@@ -0,0 +1,89 @@
+# Introduce "fish" operator (Kleisli composition)
+
+* Author: Ilya Puchka
+* Review Manager:TBD
```

Comment:
Since @inamiy has been providing good feedback, it makes sense for him to be the review manager.

Reply:
I'd like to add @dmcrodrigues as well, as he also was more involved.